app/components/organisms/ListView/styled.js

```diff
@@ -1,7 +1,5 @@
 import styled from 'styled-components';
 
-export const ListViewWrapper = styled.section`
-  max-width: 90vw;
-`;
+export const ListViewWrapper = styled.section``;
```

Comment:
The whole file can be deleted and swapped with section tag inside components

Reply:
Removed